release_notes/49-release-notes.adoc

```diff
@@ -0,0 +1,478 @@
+
+This optimization was achieved by changing how the Sensor tracks and reports updates to Central. Previously, Sensor retained full details of all open connections, endpoints, and processes when they remained active and consumed memory required to store all details. Now, Sensor stores only a fingerprint, or hash, of each object, greatly reducing memory usage. The extra CPU cost arises from computing these hashes for every object reported by Collector.
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+This release contains the following changes:
+
+//ROX-24620
+[id="image-technology-change"]
+== Product image build and release technology change
+
+If you are updating from an {product-title-short} release earlier than 4.6.10, 4.7.7, or 4.8.3, you might notice changes to container image metadata, such as container labels or Software Bill of Materials (SBOM) contents and location. This change is because product images are now built and released by using different technologies. These changes do not affect product functionality, but they might impact your third-party integrations.
+
+//ROX-29723
+[id="policy-editor-enhancements-updates_{context}"]
+=== Enhancements and updates to policy editor fields
+
+The policy editor in the {product-title-short} portal is enhanced and includes rearranged and renamed policy criteria to better reflect their intended use in policy lifecycle stages. The policy lifecycle selection process is simplified and the introductory text is updated to assist you in authoring policies. The following changes are included:
+
+* Policy criteria categories are grouped under top-level headings that reflect how the criteria trigger policy violations.
+* Criteria related to image scan results is moved out of the *Image contents* category into a new category, *Image scanning*.
+* A new category, *Baseline deviation*, is created and the *Unexpected network flow* and *Unexpected process execution* criteria are moved into it.
+* Runtime audit log policy criteria are split into *Resource operations* and *Resource attributes*.
+
+For more information, see xref:../operating/manage_security_policies/custom-security-policies.adoc#custom-security-policies[Creating and modifying security policies].
+
+//Compliance
+
+//ROX-29793
+[id="compliance-access-rights_{context}"]
+=== Permission updates for Compliance menus and API endpoints
+
+Accessing the following items now additionally requires `read` permissions for the cluster resource:
```

Comment:
RH Style > Release notes > Tenses in release notes
Another instance of "now." 

Suggest:
"With these permission updates, accessing the following items requires additional ...."